static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx

```diff
@@ -454,6 +515,23 @@ function ReleaseComparisonChart({
       {
         type: ReleaseComparisonChartType.CRASHED_SESSIONS,
         role: 'default',
+        drilldown: defined(issuesTotals?.unhandled) ? (
```

Comment:
should we create a reusable component passing the different URLs?

Reply:
For now, I think I prefer composition. We are supposed to introduce more drilldowns, so I'll revisit it then.